### PR TITLE
Improve table performance

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Prevent double render on sort change.

### CODE REVIEW

File: src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx [hunk 1]
```diff
@@ -182,11 +196,14 @@ export const FeatureToggleListTable: VFC = () => {
             tableState.search = globalFilter;
         }
 
-        setSearchParams(tableState, {
-            replace: true,
-        });
+        window.history.replaceState(
```
Comment (Tymek) [inline src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx:199]: TODO: prefix and compatibility

**Comment by FredrikOseberg** [comment]
Just to be clear: [Hot module reloading performs double rendering](https://github.com/gaearon/react-hot-loader/issues/930) by default. Are we certain that this issue exists in production code?

**Comment by Tymek** [comment]
@FredrikOseberg `react-router` adds overhead. Maybe related to: https://github.com/remix-run/react-router/issues/7634

Probably not worth optimizing more right now. (data for production build, _n=20_ samples)

**Comment by ivarconr** [comment]
[Measure and improve frontend performance](https://trello.com/c/GGF5FVao/1232-measure-and-improve-frontend-performance)

**Review by FredrikOseberg — APPROVED** [review]
I agree with your assessment. Nice discoveries.

**Pull request merged**